### ISSUE TITLE
. Erreur unexpected keyword argument 'evaluation_strategy' :

### ISSUE THREAD

**Issue by Simon-nsilulu** [googlecolab/colabtools#5332]
**Describe the current behavior**
I installed the latest version of transformers but the error described above continues to be there. So when I run the following part of the code, I get these errors even though my library imports were done correctly. I am now unable to run models like DeepPavlov and others. 

from datasets import load_dataset
from transformers import AutoTokenizer, AutoModelForSequenceClassification, Trainer, TrainingArguments
from sklearn.metrics import accuracy_score, classification_report
import torch
import pandas as pd

training_args = TrainingArguments(
            output_dir=f"./results/{model_name.replace('/', '_')}",
            evaluation_strategy="epoch",
            save_strategy="epoch",
            learning_rate=2e-5,
            per_device_train_batch_size=16,
            num_train_epochs=3,
            weight_decay=0.01,
            logging_dir="./logs",
            fp16=True
        )

        trainer = Trainer(
            model=model,
            args=training_args,
            train_dataset=train_dataset,
            eval_dataset=val_dataset,
            tokenizer=tokenizer,
            compute_metrics=lambda p: {"accuracy": accuracy_score(p.label_ids, p.predictions.argmax(axis=-1))},
        )
The code is unable to complete this step due to the following error "TrainingArguments.__init__() got an unexpected keyword argument 'evaluation_strategy". I would like to point out that two weeks ago, this code was running without any problems. I think there is a problem now.

**Describe the expected behavior**
I need you to help me complete this step and if there is really a bug, please fix it. It is inexplicable that the same code gives me more than 10 results and today it does not work.

**What web browser you are using**
(Chrome, Firefox.)

**Comment by metrizable** [comment]
@Simon-nsilulu  Thanks for filing the issue and thanks for using Colab.

Please provide repro steps in a minimal Colab notebook which is viewable by our team.
